### PR TITLE
Change link requirement into strats: Central Crateria

### PR DESCRIPTION
**Pull request by fdtaillefer**

### CODE REVIEW

File: region/crateria/central.json [hunk 1]
```diff
@@ -200,92 +238,139 @@
...
-                    "HiJump"
+                  "name": "Base",
+                  "notable": false,
+                  "requires": [
+                    {"or": [
+                      "canWalljump",
+                      "canFly",
+                      "SpeedBooster",
+                      "canDestroyBombWalls",
+                      "HiJump"
```
Comment (osse101) [inline region/crateria/central.json]: springball jump

File: region/crateria/central.json [hunk 2]
```diff
@@ -2117,29 +2248,99 @@
...
+                    {"canShineCharge": {
+                      "usedTiles": 31,
+                      "shinesparkFrames": 130,
+                      "openEnd": 2
+                    }},
+                    "canUseFrozenEnemies"
+                  ]
+                },
+                {
+                  "name": "Kill the Boyons",
```
Comment (osse101) [inline region/crateria/central.json]: I find this one hard to get anything useful out of without a shortcharge, compared to bluesuit jump

File: region/crateria/central.json (hunk 1 above)
Comment (fdtaillefer) [reply region/crateria/central.json]: This is why trying to have walljump as a tech is so much work.

File: region/crateria/central.json (hunk 2 above)
Comment (fdtaillefer) [reply region/crateria/central.json]: Hm... you're right. I feel like I could just make it notable as an alternate version, but that feels pointless.

I'll change it into a 1-tap, that'll make it a meaningful alternative.

**Pull request merged**